ui/widgets/nuxeo-directory-radio-group.js

```diff
@@ -150,7 +150,13 @@ import { DirectoryWidgetBehavior } from './nuxeo-directory-widget-behavior.js';
     }
 
     _isChecked(entry) {
-      const value = (this.value && this.value.length > 0 ? this.value : null) || this.selectedItem;
+      let value;
+      if ((this.value && this.value.length > 0) || typeof this.object === 'object') {
```

Comment:
Given that this.value is of type `String`, in what situation do we need to do `this.value && this.value.length > 0` instead of `this.value`? Also, where does this `this.object` come from?